cinema_service/settings.py

```diff
@@ -20,14 +24,13 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = os.environ.get("SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = os.environ.get("DEBUG")
```

Comment:
The DEBUG variable should be cast to a boolean value. os.environ.get() returns a string, which could lead to unexpected behavior. You can use the strtobool() function from the distutils.util module to convert the string to a boolean.